pkg/koordlet/resmanager/be_reconcile.go

```diff
@@ -0,0 +1,361 @@
+/*
+ Copyright 2022 The Koordinator Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package resmanager
+
+import (
+	"io/ioutil"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/audit"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/statesinformer"
+	"github.com/koordinator-sh/koordinator/pkg/util"
+	"github.com/koordinator-sh/koordinator/pkg/util/system"
+)
+
+func (r *resmanager) reconcileBECgroup() {
+	nodeSLO := r.getNodeSLOCopy()
+	podsMeta := r.statesInformer.GetAllPods()
+	for _, podMeta := range podsMeta {
+		if extension.GetPodQoSClass(podMeta.Pod) != extension.QoSBE {
+			continue
+		}
+		if getCPUSuppressPolicy(nodeSLO) != v1alpha1.CPUCfsQuotaPolicy {
+			reconcileBECPULimit(podMeta)
```

Comment:
extend resource requires request==limit, so we should support limit.batch-cpu and limit.batch-memory on annotation field.
https://github.com/koordinator-sh/koordinator/issues/53

Reply:
open a new issue to follow up on this issue

Reply:
> open a new issue to follow up on this issue

#83